### ISSUE TITLE
Tab Order for "Replace" doesn't work

### ISSUE THREAD

**Issue by robpaveza** [Microsoft/vscode#5171]
- VSCode Version: 0.10.6
- OS Version: Windows 10 10.0.10240.0

Steps to Reproduce:

1. Have a text file open
2. Press Ctrl+H
3. The "Replace" text box is in focus
4. Press "Tab" at least 3 times.

Observed behavior: There is no way to get to the "Find" box from just tab order.  It occurred to me that Ctrl+F will get there, but that's definitely not obvious and not in keeping with usual mechanisms.

**Comment by alexandrudima** [comment]
@robpaveza It is an accessibility requirement that Tab is not "trapped" in widgets.

The only widget we have that traps "Tab" is the editor itself and that can be toggled via Ctrl+M.

To get from the Replace text box to the Find text box, I use Shift-Tab or Ctrl+F.
